public/app/features/plugins/importer/importPluginModule.ts

```diff
@@ -67,17 +67,31 @@ export async function importPluginModule({
   }
 
   return SystemJS.import(modulePath).catch((e) => {
-    let error = new Error('Could not load plugin: ' + e);
-    console.error(error);
-    pluginsLogger.logError(error, {
-      path,
-      pluginId,
-      pluginVersion: version ?? '',
-      expectedHash: moduleHash ?? '',
-      loadingStrategy: loadingStrategy.toString(),
-      sriChecksEnabled: String(Boolean(config.featureToggles.pluginsSriChecks)),
-      newPluginLoadingEnabled: String(Boolean(config.featureToggles.enablePluginImporter)),
-    });
-    throw error;
+    // If rendererDisableAppPluginsPreload is enabled log a specific error message
+    if (config.featureToggles.rendererDisableAppPluginsPreload) {
```

Comment:
Should we also check if it is the renderer that initiated the request ([like here](https://github.com/grafana/grafana/blob/ca63c8015d45396ce300cce365b3ecb1b6797d48/public/app/app.ts#L238))?